### PR TITLE
Feature/3897 form validation required

### PR DESCRIPTION
**Pull request by joannasia**
## Description

- Error Dot for valid section, some errors validation

**New stuff** ✨

* ErrorDot component

(Resolves | Contributes to) #3897

### CODE REVIEW

File: src/modules/dashboard/components/ExpenditurePage/TitleDescriptionSection/TitleDescriptionSection.css
Comment (jakubcolony) [inline src/modules/dashboard/components/ExpenditurePage/TitleDescriptionSection/TitleDescriptionSection.css:37]: Could you please explain which element you're trying to target here?

**Review by jakubcolony — CHANGES_REQUESTED** [review]
Hey @joannasia, great work with the errors! 
image
image

I'm not sure if it was introduced in this PR but I noticed the spacing between "Value is required" message and bottom border should be 7 or 8px according to Figma, whereas it currently looks like it's 0:
image

File: src/modules/dashboard/components/ExpenditurePage/Streaming/SingleFundingSource.tsx
Comment (joannasia) [reply src/modules/dashboard/components/ExpenditurePage/Streaming/SingleFundingSource.tsx]: yep, changed it

File: src/modules/dashboard/components/ExpenditurePage/Payments/SingleRecipient/SingleRecipient.tsx
Comment (joannasia) [reply src/modules/dashboard/components/ExpenditurePage/Payments/SingleRecipient/SingleRecipient.tsx:59]: I have fixed it everywhere the wrong spelling appeared

File: src/modules/dashboard/components/ExpenditurePage/Payments/SingleRecipient/SingleRecipient.tsx
Comment (joannasia) [reply src/modules/dashboard/components/ExpenditurePage/Payments/SingleRecipient/SingleRecipient.tsx]: thanks, fixed

File: src/modules/dashboard/components/ExpenditurePage/Payments/Payments.css
Comment (joannasia) [reply src/modules/dashboard/components/ExpenditurePage/Payments/Payments.css]: it is fixed

Comment (joannasia) [reply src/modules/dashboard/components/ExpenditurePage/Payments/Payments.css]: @jakubcolony when it comes to the errors for the value i have changed it to the 12 px, but for the rate there is no space to give it more padding: below screen
@arrenv can the error for the field stay like that or there are some additional style improvements needed ?
Screenshot 2022-10-07 at 11 01 31

File: src/modules/dashboard/components/ExpenditurePage/TitleDescriptionSection/TitleDescriptionSection.css
Comment (joannasia) [reply src/modules/dashboard/components/ExpenditurePage/TitleDescriptionSection/TitleDescriptionSection.css:37]: i wanted to get to the child component node in this place

Screenshot 2022-10-07 at 11 01 31

Comment (jakubcolony) [reply src/modules/dashboard/components/ExpenditurePage/TitleDescriptionSection/TitleDescriptionSection.css:37]: Makes sense, thanks. You might want to consider adding a comment here

**Review by jakubcolony — APPROVED** [review]
Hey @joannasia, apologies for the delay, I was off on Monday. 
Thanks for making the requested changes, this looks good to me now!

**Comment by willm30** [comment]
@joannasia FYI, we generally wait for two approving reviews before merging.

**Comment by joannasia** [comment]
@willm30 oh sorry, didn't know about it, thank you for the information, i'll wait in next PRs

**Pull request merged**
